wallet/src/main/java/com/android/identity_credential/wallet/WalletApplication.kt

```diff
@@ -62,5 +71,26 @@ class WalletApplication : Application() {
         issuingAuthorityRepository = IssuingAuthorityRepository()
         issuingAuthorityRepository.add(SelfCertificationIssuingAuthority(this, storageEngine))
         issuingAuthorityRepository.add(PassportBasedIssuingAuthority(this, storageEngine))
+
+        initTrustManager()
+    }
+
+    private fun initTrustManager(){
+        // init Trust Manager
+        // read TrustPoint owf pem cert to manually add an "owf identity credential reader" (appverifier)
+        val owfIdentityCredentialReaderRoot = Certificate.fromPem(
+            String(
+                resources
+                    .openRawResource(R.raw.owf_identity_credential_reader_cert)
+                    .readBytes()
+            )
+        );
+        trustManager.addTrustPoint(
+            TrustPoint(
+                owfIdentityCredentialReaderRoot.javaX509Certificate,
+                displayName = "OWF Identity Credential Reader",
+                null
```

Comment:
I've added it as an extension from the Wallet app but I think this function should live within `TrustManager` itself overloading the existing function `addTrustPoint()`. Because the extension function accepts all the parameters required to create a `TrustPoint` (no wallet-specific data is passed) this task could easily belong to `TrustManager` which works to manage `TrustPoints`.

Reply:
`TrustManager` lives in the `identity` library which cannot have any Android dependencies so unfortunately that's not possible.